### PR TITLE
feat: jose/robusta nightly

### PR DESCRIPTION
**Pull request by jrmanes**
Hello!

This PR contains the resources for running tests against Robusta
It has been tested and it works, we can connect to robusta-nightly 👌 

Also, it contains the changes pulled from the PR: https://github.com/celestiaorg/test-infra/pull/208 (*maybe we'll have to wait until merge it*)

Best!

Jose Ramon Mañes

### CODE REVIEW

**Comment by jrmanes** [comment]
keep it open until we merge: https://github.com/celestiaorg/test-infra/pull/208

File: compositions/cluster-k8s/robusta/flood.toml [hunk 1]
```diff
@@ -40,24 +39,24 @@
...
 [[groups]]
-id = "fulls"
-builder = "docker:generic"
+  id = "fulls"
+  builder = "docker:generic"
 [groups.resources]
-memory = "8000Mi"
-cpu = "6000m"
+  memory = "200Mi"
+  cpu = "200m"
```
Comment (Bidon15) [inline compositions/cluster-k8s/robusta/flood.toml:46]: I would give full nodes more resources 
like 4gi and 2000m at least

File: compositions/cluster-k8s/robusta/flood.toml [hunk 2]
```diff
@@ -15,21 +15,20 @@
       execution-time = "20"
       light = "140"
```
Comment (Bidon15) [inline compositions/cluster-k8s/robusta/flood.toml:16]: lights should be the same figure as in groups

**Comment by jrmanes** [comment]
hello @Bidon15 
I've resolved some conflicts that we had after merging the other PR.
Let me know when you'll check it if you find anything to modify :)

cheers!

**Review by Bidon15 — APPROVED** [review]

**Pull request merged**